### DISCUSSION TITLE
How to install last version package?

### DISCUSSION THREAD

**Discussion by vanarok** [termux/termux-packages#18459] [Q&A] [answered]
I need version (0.20.0) https://github.com/termux/termux-packages/commit/e99ec6473b570e6012eaf00674c0435671cf34cb
, but installed (0.20.0-alpha.4-0)
![Screenshot_2023-11-12-11-35-04-536_com termux](https://github.com/termux/termux-packages/assets/6105977/4e48ef2c-c105-4bae-93df-5125894728fe)

**Comment by truboxl** [comment]
It looks like 0.20.0 didn't get uploaded or ignored, https://packages-cf.termux.dev/apt/termux-main/pool/main/d/distant/

Either need to manual clean up from server and reupload the debs or rebuild with a fake higher version eg: 0.20.0.0, CC @sylirre @Grimler91

**Reply by sylirre** [reply, marked as answer]
@truboxl Need to bump package epoch. Otherwise users won't be able to upgrade.

https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#downgrading-the-package-or-changing-versioning-scheme
